### ISSUE TITLE
Navigating back to parent directory in file browser is not possible with keyboard.

### ISSUE THREAD

**Issue by clapierre** [jupyterlab/jupyterlab#6580]
From Web4All Hackathon https://github.com/diagram-codesprint/jupyterlab/issues/5
**Describe the bug**
After navigating to a sub-directory, I cannot navigate back to the parent directory using the back button or arrow keys.  This is an accessibility issue for users who only use the keyboard.

**To Reproduce**
1. Navigate to the file browser.
2. Navigate to a sub-directory.
3. There's no way to navigate back to the parent directory with keyboard.

**Expected behavior**
I should be able to use either the left arrow or backspace to navigate back up to the parent directory.

**Desktop (please complete the following information):**

- OS: macOS Mojave (10.14.4)
- Browser Chrome 4.0.3729.131
- JupyterLab 1.0.0a3

**Comment by orena1** [comment]
Also, a mouse with a back button:
![image](https://user-images.githubusercontent.com/8983713/100261416-60566d80-2f53-11eb-9546-1caebca9ebac.png)

 does not work, instead of going one folder up it goes to the previous chrome page

**Comment by krassowski** [comment]
This was also reported in #6808 and fixed by https://github.com/jupyterlab/jupyterlab/pull/6859.
